docs/modules/sql/pages/mapping-to-jdbc.adoc

```diff
@@ -36,7 +36,7 @@ Before you can create a mapping to an external data store, you must have the fol
 
 - A JDBC driver that's compatible with your data store. This driver must be on the classpath of your cluster members:
 ** The full distribution of Hazelcast comes with a JDBC driver for MySQL, PostgreSQL, and H2 data stores.
-** The slim distribution of Hazelcast does not come with a JDBC driver.
+** The slim distribution of Hazelcast and Hazelcast Enterprise do not come with a JDBC driver.
```

Comment:
Slim has no drivers. Full distribution has drivers with the exception of MySQL driver for Hz Enterprise - this driver we couldn't include due to licensing issue (we've excluded whole `hazelcast-cdc-mysql` module).

Reply:
Updated as requested. I've left the slim distribution line as in the original because we're specifically talking about the JDBC driver in this topic.